### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by MephistoDevelop**
CR#1

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
```
Comment (efrapp) [inline config/routes.rb:1]: Please add the `root` to show up the login page.

Comment (efrapp) [reply config/routes.rb:1]: Also, Add all the needed routes for users and posts.

**Review by efrapp — CHANGES_REQUESTED** [review]
# We need a step further! 🚴 

Hi guys, how are you? You are doing a good job so far, however, there are some missing parts in your project. I'm curious about why you didn't add the controllers for users and posts? Those are required in this milestone, the same applies to the views. Another thing, you need to use Rspec instead of Minitest, this is a requirement from the Association project and it should be applied here too. 

**Required changes**
- [x] Stickler should have validations in green
- [x] Testing should be with `Rspec` + `Capybara`
- [x] User can sign-in
- [x] UsersController#index displays all users
- [x] PostsController#index displays posts in a timeline
- [x] User can create posts
- [x] Post is always displayed with content and author

**Optional changes**
- [ ] Add `.ruby-version` file to know what is the Ruby version you are using in the project.

Please, submit a new code review with the changes mentioned.

**Status:** `Changes requried`

**Review by bellom — APPROVED** [review]
Hi Team,
**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in your next milestone :slightly_smiling_face:_ 

Best,

Oluwaseun J. Bello

**Pull request merged**